src/styles/learn.scss

```diff
@@ -72,6 +72,11 @@
   }
 }
 .jf-article-page {
+  .jf-summary-box {
+    p {
+      line-height: 120%;
```

Comment:
i'll add this to the shortcut card, but for thinking over later: maybe we should have a class for 120% line height? or else a class for paragraph text with 120% height...